Automattic/wp-calypso
# Jetpack Social: Update copy for connections page

#### Proposed Changes

* This PR adds some simple copy changes for the connections page for Jetpack Social, taking better advantage of the available space.

#### Testing Instructions



* Open the live links in both Calypso Blue and Green.
* Make sure the copy changes look good, and match the screenshot.

<img width="1167" alt="image" src="https://user-images.githubusercontent.com/1713699/210900343-14ba0c19-fa74-4198-af21-4ee3c68f5e30.png">


#### Pre-merge Checklist
